src/docs/products.ts

```diff
@@ -23,6 +23,24 @@ export const productSchema = {
     } 
 }
 
+export const searchParams = {
+    properties: {
+        type:"params",
+        name: {
+            type: "string"
+        },
+        minPrice: {
+            type: "number"
+        },
+        maxPrice: {
+            type: "number"
+        },
+        category: {
+            type: "string"
+        },
+    }
+}
+
 export const getProducts = {
```

Comment:
are you using this?

Reply:
No, I'm not using it. I forgot to delete